docs/learn/wallets/firefly-wallet.md

```diff
@@ -4,32 +4,164 @@ description: The IOTA Firefly Wallet sets a new standard for Software Wallets in
 ---
 
 
-# Heading 1 (Page Title)
+# The Firefly Wallet
 
-content - introduction to the topic
+The Secure entry point to the IOTA Ecosystem.
 
+Firefly is built with the future in mind. It will evolve into a tool that offers much more than being just a Software Wallet for your IOTA Tokens.
+Firefly has a modular design that makes it easy to add innovative new features later on.
+IOTA's new wallet guides you through the complexities of crypto, protected and with ease, while it is using cutting edge technology to secure your tokens.
 
-## Heading 2 (Sub topic 1)
+Some of the most important features:
+- Written in Rust
```

Comment:
Hey Jeroen. Very much agree with you. The problem with Firefly is... there is actually no Docu atm. The Team is totally blocked with work... 
But the Wallet should have and needs a propper description... It should have been shipped out with the product actually... So i am going a bit over our scope here and think this needs to be covered if the topic is not covered by the maintainers yet... Maybe they even wanna take over the content later - easy to remove most from here and port it in the Firefly documentation. Lets say for something like Hornet - we of course dont need do do this level of detail... but for other things it is better to do it here instead of having nothing.
I also think that some projects miss a simple approach to the topic... they give a detailed, but complicated description... This wont help users to understand the tech... If we find such issues it can be on us to provide a better approach on some things... but surely shouldnt happen to often...

Reply:
Ah cool! No problem, just trying to wrap my head around what exactly we are going to provide and what not to prevent doing double work. But if Firefly docs are in the state like you describe, your content is super useful!

With regard to the implementation details I stick to my point though, I would rather not see that on the Wiki. While it is awesome tech, it distracts from what should be an easy guide to how to use Firefly.

Reply:
Yeah good point - thats taken from Firefly Website... I started with grabbing content from IF sources on this content and then really recognised that we dont have any description or "user guide" for Firefly... As i have a kind of little "personal connection" to Firefly as it was the first project for me where i really contributed and i think i have a very good knowledge from the beat test i took this part on me and will try to write a real user guide to Firefly here... Definitely much more as it will be on other topics...